docs/component-guidance/inputter.md

```diff
@@ -0,0 +1,251 @@
+### Anatomy of the component
+
+![Anatomy of the inputter](../images/inputter/inputter-anatomy.png)
+
+#### Label
+
+Every single `muon-inputter` must have a label to identify the form control and should describe its purpose – which helps users understand. See https://www.w3.org/WAI/tutorials/forms/labels/
+
+#### Input area
+
+The section a user interacts with to enter information or indicate a preference. It could be a text box, radio button, selector or checkbox etc.
+
+Example of an input area with data added:
+
+![Input area and data](../images/inputter/inputter-input-area.png)
+
+#### Helper text
+
+Each `muon-inputter` can have optional helper text added for additional in-context assistance. This is a short sentence that appears below the label in the hierarchy of elements and its purpose is to help the user fill in the input.
+
+#### Tip detail
+
+Each `muon-inputter`'s helper can have an optional tip added for further detail and in-context assistance. This is a short sentence that appears below the helper in the hierarchy of elements and is always used in conjunction with the helper.
+
+#### Headings
+
+![Headings](../images/inputter/inputter-headings.png)
+
+This attribute is used for a group of checkboxes or radio buttons, where the label is adjacent to each `muon-inputter` checkbox or radio button.
+
+#### Validation
+
+![Validation](../images/inputter/inputter-validation.png)
+
+This is used for error prevention. It checks the format of the inputted content against the required criteria. By default, it uses the minimal `isRequired`, but can vary depending on the `muon-inputter` type – for example, an Email type requires the format to contain an '@' symbol etc. The validations are optional and can be configured to requirements.
+
+#### Placeholder
+
+![Placeholder](../images/inputter/inputter-placeholder.png)
+
+This provides the user with an example of the expected input.
+
+#### Masks and separators
+
+![Masks and seperators](../images/inputter/inputter-masks-seperators.png)
+
+A mask can be used to show what can be entered. A separator is used with the mask to manipulate the inputted value if it needs to be in a specific format, such as a bank sort code.
+
+### Number min/max
+
+The Number type of `muon-inputter` can be formatted to have a minimum and maximum value. For example the minimum might be 0 and the maximum 10 (by default) so the user cannot enter a value higher or lower than this, nor can they increase/decrease the value outside of the range by using the arrows that appear when interacting with it.
+
+---
+
+### Types
+
+This is a very versatile and expansive component used in forms – there are 15 types of `muon-inputter`:
```

Comment:
These are not types, but just different inputs. The inputter comes with 2 types standard and reset

Reply:
```suggestion
This is a very versatile and expansive component used in forms – there are 15 different inputs that can be chosen from within `muon-inputter`:
```